src/converter/modules/blocks/create-guides-list.js

```diff
@@ -1,14 +1,29 @@
+import yaml from 'js-yaml';
 import { toBlock, replaceElement } from '../utils/dom-utils.js';
 
-export default function createGuidesList(document) {
-  const ullistElement = document.querySelector('main > div > ul');
+export default function createGuidesList(document, meta) {
+  const ullistElements = Array.from(
+    document.querySelectorAll('main > div > ul'),
+  );
+  const fullMetadata = yaml.load(meta);
+  if (fullMetadata.type === 'Documentation') {
+    if (ullistElements.length) {
+      ullistElements.forEach((ullistElement) => {
+        const prevElement = ullistElement.previousElementSibling; // it can be either h2 or h3
+        const h2Element = prevElement.previousElementSibling;
+        if (
+          (prevElement.tagName === 'H2' &&
+            prevElement.id === 'lists-documentation') ||
+          h2Element.tagName === 'H2'
+        ) {
```

Comment:
Are we not checking the prevElement Tag name with H3 comparison? As per the comment "it can be either h2 or h3"

Reply:
We need only H2s. In case there is a H3 before <ul>, i am again checking the previous element which is always h2.
const prevElement = ullistElement.previousElementSibling; // it can be either h2 or h3
const h2Element = prevElement.previousElementSibling;

Reply:
Removed the additional check as those kind of pages that do not have direct h2 before <ul> are not yet accessible.